model_compression_toolkit/gptq/keras/gptq_training.py

```diff
@@ -169,16 +162,17 @@ def get_activation_quantizer_holder(self, n: common.BaseNode) -> Union[None, Cal
         Returns:
             A ActivationQuantizationHolder layer for the node activation quantization.
         """
-        if self._is_gptq_applicable(n): # If it was wrapped before - add holder instead
-            _, activation_quantizers = quantization_builder(n, self.gptq_config)
-            # Holder by defenition uses a single quantizer for the activation quantization
-            # thus we make sure this is the only possible case (unless it's a node we no activation
-            # quantization, which in this case has an empty list).
-            if len(activation_quantizers) > 0:
-                assert len(activation_quantizers) == 1, f'ActivationQuantizationHolder supports a ' \
-                                                        f'single quantizer but {len(activation_quantizers)} quantizers were found for node {n}'
-                return ActivationQuantizationHolder(activation_quantizers[0])
-        return None
+        _, activation_quantizers = quantization_builder(n, self.gptq_config)
+        # Holder by definition uses a single quantizer for the activation quantization
+        # thus we make sure this is the only possible case (unless it's a node we no activation
+        # quantization, which in this case has an empty list).
+        if len(activation_quantizers) > 0:
+            assert len(activation_quantizers) == 1, f'ActivationQuantizationHolder supports a ' \
```

Comment:
Why not logger error?
Also, I would change it to an if-else statement:
`if len == 0
elif len == 1
else (raise error)`

this is clearer in my opinion